Makefile

```diff
@@ -44,7 +44,7 @@ download-kubeconfigs:
 	read SHOOT; \
 	echo "enter cluster provider(gcp|aws|azure|vsphere|openstack|alicloud|metal|equinix-metal)"; \
 	read PROVIDER; \
-	./hack/local_setup.sh --SEED $$SEED --SHOOT $$SHOOT --PROJECT $$PROJECT --PROVIDER $$PROVIDER
+	./hack/local_setup.sh --seed $$SEED --shoot $$SHOOT --project $$PROJECT --provider $$PROVIDER
 
 .PHONY: local-mcm-up
 local-mcm-up: download-kubeconfigs
```

Comment:
I don't see any usage of `local_restore.sh`. I suppose you wanted to use it in `make local-mcm-down` ?

Reply:
should i create a separate make target for this?
Previously what this script does was only mentioned as comments in local_setup.sh. Now its a separate script.

Reply:
we have a make target by the name `local-mcm-down` for what this script is doing

Reply:
As discussed i have now made changes. PTAL.